src/Components/SecondSection/Countdown/Countdown.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styles from './Countdown.module.css';
+
+const Countdown = ({ days, hours, minutes, seconds }) => {
+  return (
+    <div className={styles.group}>
+      <div className={`${styles.semiClone} ${styles.semiClone2}`}>
+        <div className={`${styles.ellipse} ${styles.ellipse19}`}></div>
+        <div className={`${styles.ellipse} ${styles.ellipse20}`}></div>
+      </div>
+      <div className={`${styles.semiClone} ${styles.semiClone1}`}>
+        <div className={`${styles.ellipse} ${styles.ellipse18}`}></div>
+        <div className={`${styles.ellipse} ${styles.ellipse19}`}></div>
+      </div>
+      <div className={`${styles.semiClone} ${styles.semiClone3}`}>
+        <div className={`${styles.ellipse} ${styles.ellipse21}`}></div>
+        <div className={`${styles.ellipse} ${styles.ellipse22}`}></div>
+      </div>
```

Comment:
What are these?
I have tried to comment them but I cannot tell what difference they make.

Reply:
I have improved the code. These lines represent the two points between the counter.